<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginDialog</class>
 <widget class="QDialog" name="CLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="pNameCBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>139</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pPwdEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pLoginButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pAutoLgnCBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>199</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登陆</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pRmbPwdCBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>199</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pRegUserBtn">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>142</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>注册用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pFgtPwdBtn">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>172</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
